homeassistant/components/solaredge_local/sensor.py

```diff
@@ -167,6 +253,15 @@ def unit_of_measurement(self):
         """Return the unit of measurement."""
         return self._unit_of_measurement
 
+    @property
+    def device_state_attributes(self):
+        """Return the state attributes."""
+        if self._attr:
+            try:
+                return {self._attr: self._data.info[self._json_key]}
+            except KeyError:
+                return None
+
```

Comment:
We still need a return at the end for the other case.
```suggestion
        return None
```